index.js

```diff
@@ -2,9 +2,10 @@ const api = require('./sf_api')
 const fsUtils = require('./fs_utils')
 const fs = require('fs');
 
+const RECONCILIATOIN_FIELDS_TO_SYNC = ["name_nl", "name_fr", "name_en", "auto_hide_formula", "text_configuration", "virtual_account_number", "reconciliation_type", "public", "allow_duplicate_reconciliations", "is_active", "tests"]
```

Comment:
`allow_duplicate_reconciliations`, `is_active` and `public` aren't exported in our API

Reply:
I created a branch on the monolith but forgot to open the MR
It's open now https://gitlab.silverfin.com/development/silverfin/-/merge_requests/15429